v5/core/unmarshal_v2_models_test.go

```diff
@@ -173,6 +173,11 @@ func TestUnmarshalModelInstance(t *testing.T) {
 	assert.NotNil(t, myModel)
 	assert.Equal(t, "string1", *(myModel.Foo))
 	assert.Equal(t, int64(44), *(myModel.Bar))
+
+	// Unmarshal with "zero" input should return an error
+	var rawMap2 map[string]json.RawMessage
+	err = UnmarshalModel(rawMap2, "", &myModel, UnmarshalMyModel)
+	assert.NotNil(t, err)
```

Comment:
we should probably also have a test that exercises an un-initialized []RawMessage as well

Reply:
Added.